database/drivers/driver.go

```diff
@@ -0,0 +1,20 @@
+package drivers
+
+import (
+	"log"
+
+	"gnorm.org/gnorm/database"
+)
+
+// Driver defines the base interface for databases that are supported by gnorm
+type Driver interface {
```

Comment:
Let's move this interface into gnorm/database.... there's no need for this to be in a separate package since it specifically references database.info.  The drivers directory honestly doesn't need to exist, but I kind of like it organizationally.